projects/torvalds/linux/expected_artifacts

```diff
@@ -8,4 +8,3 @@ bootconfig-linux-amd64-$GIT_TAG.tar.gz
 bootconfig-linux-amd64-$GIT_TAG.tar.gz.sha256
 bootconfig-linux-amd64-$GIT_TAG.tar.gz.sha512
 githash
-linux/linux-amd64/tools/bootconfig
```

Comment:
i took this out because we dont usually do this, we just push the tars. was there a good reason we also needed the raw binary on s3?